src/app/pages/system/advanced/sessions/sessions-card/sessions-card.component.ts

```diff
@@ -108,22 +68,66 @@ export class SessionsCardComponent {
     private loader: AppLoaderService,
     private ws: WebSocketService,
     private advancedSettings: AdvancedSettingsService,
-  ) {}
+    private cdr: ChangeDetectorRef,
+  ) {
+    this.updateSessions();
+  }
+
+  updateSessions(): void {
+    this.isLoading$.next(true);
+    this.ws.call('auth.sessions', [[['internal', '=', false]]]).pipe(
+      map((sessions) => this.sessionsSourceHelper(sessions)), untilDestroyed(this),
```

Comment:
I'm not a huge fan of doing transformations like this. You need a new interface and it tends to get messy. I'd rather we keep original interface and format data in table-related code.